package.json

```diff
@@ -1,34 +1,25 @@
 {
-  "name": "release-with-changelog",
-  "version": "1.0.0",
-  "description": "Create a GitHub relase with changelog for every tag you push",
-  "main": "distribution/index.js",
-  "scripts": {
-    "package": "ncc build index.js -o distribution",
-    "test": "xo"
-  },
-  "repository": "notlmn/release-with-changelog",
-  "keywords": [
-    "github",
-    "actions",
-    "javascript",
-    "tags",
-    "releases",
-    "changelog"
-  ],
-  "author": "Laxman Damera",
-  "license": "MIT",
-  "dependencies": {
-    "@actions/core": "^1.2.6",
-    "@actions/github": "^4.0.0"
-  },
-  "devDependencies": {
-    "@zeit/ncc": "^0.22.3",
-    "xo": "^0.32.1"
-  },
-  "xo": {
-    "ignore": [
-      "distribution"
-    ]
-  }
+	"private": true,
+	"scripts": {
+		"package": "ncc build index.js -o distribution",
+		"test": "xo && npm run jest",
+		"jest": "GITHUB_SERVER_URL='https://github.com' GITHUB_REPOSITORY='test/test' jest"
```

Comment:
Since these are part of the tests, why not just specify them in the test file?

Reply:
Set it in beforeAll https://github.com/vuejs/vue-test-utils/issues/193#issuecomment-528358591

Reply:
That still doesn't seem to fix it, tried `beforeAll` as well.

![image](https://user-images.githubusercontent.com/37769974/96019131-bbcc1080-0e69-11eb-97cb-8a25652c2ac1.png)

Reply:
Got it, because env has to be set before doing `require('./generate-release-notes')`, as `process.env` is access at the top of that module.

Reply:
Indeed I usually solve that by importing a fixtures.js file, so it’s executed before other imports